graph/src/data/store/mod.rs

```diff
@@ -188,8 +182,6 @@ impl Value {
                     .ok_or_else(|| QueryExecutionError::NamedTypeError("Int".to_string()))?
                     as i32,
             ),
-            (query::Value::Variable(var), _) => Value::String(var.to_owned()),
```

Comment:
Is this not needed?

Reply:
No, we should have resolved variables by now.

Reply:
Ah, yes 👍